js/okex.js

```diff
@@ -752,8 +752,8 @@ module.exports = class okex extends Exchange {
             baseId = this.safeString (parts, 0);
             quoteId = this.safeString (parts, 1);
         }
-        const inverse = baseId === settleCurrency;
-        const linear = quoteId === settleCurrency;
+        const inverse = contract ? baseId === settleCurrency : undefined;
```

Comment:
Let's always wrap expressions in ternary conditionals with brackets:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#using-ternary-conditionals

Reply:
```JavaScript
const inverse = contract ? (baseId === settleCurrency) : undefined;
```